operations/kube-manifests/ConfigMap-tempo-compactor.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
```

Comment:
Why this file is called Compactor configMap?

Reply:
I'm not sure I understand. This is indeed the configmap that the compactor uses..